nagios/nagios-metrics-wf.py

```diff
@@ -112,35 +113,75 @@ def remove_trailing_dots(text):
     parser.add_argument('--service', dest='service', action='store_true',
                         help='proccess "service-perfdata" (default: "host-perfdata")')
     parser.add_argument('--test', dest='test', action='store_true')
-    parser.add_argument('--wf_server', dest="server")
-    parser.add_argument('--wf_token', dest="token")
-    parser.add_argument('--wf_proxy_addr', dest="addr")
-    parser.add_argument('--wf_proxy_port', dest="port")
+    parser.add_argument('--wf_server', dest="server", help='Wavefront Server URL.')
+    parser.add_argument('--wf_token', dest="token", help='Wavefront token.')
+    parser.add_argument('--wf_proxy_addr', dest="addr", help='Wavefront Proxy address.')
+    parser.add_argument('--wf_proxy_port', dest="port", help='Wavefront Proxy port.')
+    parser.add_argument('--csp_base_url', dest= "csp_base_url", help='Base URL of CSP console.')
+    parser.add_argument('--csp_api_token', dest="csp_api_token", help='CSP API Token for csp authentication.')
+    parser.add_argument('--csp_app_id', dest="csp_app_id", help='CSP OAuth App ID for csp authentication.')
+    parser.add_argument('--csp_app_secret', dest="csp_app_secret", help='CSP OAuth App secret for csp authentication.')
+    parser.add_argument('--csp_org_id', dest="csp_org_id", help='CSP Organization ID for csp authentication.')
```

Comment:
help='Long ID of the VMware Cloud organization running the service. The OAuth app must be added to this organization.'